src/patterns/correct-errors/example-error-proto-multiple-error-per-input.njk

```diff
@@ -0,0 +1,96 @@
+
+{% from "components/date-input/_macro.njk" import onsDateInput %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/list/_macro.njk" import onsList %}
+
+{% call
+        onsPanel({
+            title: "There is a problem with this page",
+            type: "error"
+        })
+    %}
+
+        {{
+            onsList({
+                "element": "ol",
+                "itemsList": [
+                    {
+                        "text": "Enter a number between 1 to 12",
+                        "url": "#month-error",
+                        "variants": "inPageLink"
+                    },
+                    {
+                        "text": "Enter a date that is after 1 January 2019",
+                        "url": "#year-error",
+                        "variants": "inPageLink"
+                    }
+                ]
+            })
+        }}
+    {% endcall %}
+{{
+    onsDateInput({
+        "id": "period-from-date",
+        "legendOrLabel": "Period from:",
+        "description": "For example, 31 3 2019",
+        "day": {
+            "label": {
+                "text": "Day"
+            },
+            "name": "day",
+            "value": "31"
+        },
+        "month": {
+            "label": {
+                "text": "Month"
+            },
+            "name": "month",
+            "value": "13"
+        },
+        "year": {
+            "label": {
+                "text": "Year"
+            },
+            "name": "year",
+            "value": "2018"
+        }
+    })
+}}
```

Comment:
To show the functionality I think we only need one date input and just have it ask for date of birth or something would make it simpler as the extra date input isn't really adding anything. Or could just use a single regular input with 2 problems with the answer might be even better as this isn't only going to be used on components with more than one input

Reply:
The example now no longer makes sense because you have a period from but no period to.
I think an example based on this would be good as its a real eQ example of what would happen in a real questionnaire.

<img width="699" alt="Screenshot 2023-09-28 at 13 47 00" src="https://github.com/ONSdigital/design-system/assets/42928680/fdd6a60a-5741-4409-8a52-346479333615">